src/main/java/DenisCamacho/task_01/Menu.java

```diff
@@ -0,0 +1,59 @@
+package main.java.DenisCamacho.task_01;
+
+import java.util.Scanner;
+
+public class Menu {
+
+    public static void main(String[] args) {
+        ContactManager myContactManager = new ContactManager();
+        boolean status = false;
+        do {
+            System.out.println("Menu \n 1 -> Show contact \n 2 -> Add contact \n 3 -> Search contact \n 4 -> " +
+                    "Delete contact \n 5 -> Exit");
+            Scanner options = new Scanner(System.in);
+            System.out.println("What would you like do?");
+            String option = options.nextLine();
+            switch (option) {
+                case "1":
+                    myContactManager.showContact();
+                    break;
+                case "2":
+                    System.out.println("Please into the data \n *name \n number \n email");
+                    String data = options.nextLine();
+                    while (data.equals("")) {
+                        System.out.println("Into the name please");
+                        data = options.nextLine();
+                    }
+                    Contact newContact = new Contact(data);
```

Comment:
@denis1729 What do you think using a constructor with the name, email and phone to initialize all this attribute with it, just to avoid lines using the setters